README.md

```diff
@@ -35,371 +35,33 @@ Find the helm charts and controller images on Amazon ECR Public Gallery.
-# Ensure cluster has compute
-kubectl get nodes
-```
+For a step-by-step tutorial head over to [Machine Learning with the ACK SageMaker Controller](https://aws-controllers-k8s.github.io/community/docs/tutorials/sagemaker-example/)
 
-#### 2.1 Setup IRSA for controller pod
+### 2.0 ACK Application Auto Scaling Controller
 
-Before you can deploy your operator using an IAM role, associate an OpenID Connect (OIDC) provider with your role to authenticate with the IAM service.
+For a step-by-step tutorial on how to use Application Auto Scaling Controller with SageMaker head over to [Scale SageMaker Workloads with Application Auto Scaling](https://aws-controllers-k8s.github.io/community/docs/tutorials/autoscaling-example/)
 
-##### 2.1.1 Create an OpenID Connect Provider for Your Cluster
-
-```sh
-eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
---region ${AWS_DEFAULT_REGION} --approve
-```
-
-Get the OIDC ID
-```sh
-export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
-export OIDC_PROVIDER_URL=$(aws eks describe-cluster --name $CLUSTER_NAME --region $AWS_DEFAULT_REGION \
---query "cluster.identity.oidc.issuer" --output text | cut -c9-)
-```
-
-##### 2.1.2 Create an IAM Role
-
-Create a file named trust.json and insert the following trust relationship code block required for IAM role into it.
-```sh
-printf '{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Principal": {
-        "Federated": "arn:aws:iam::'$AWS_ACCOUNT_ID':oidc-provider/'$OIDC_PROVIDER_URL'"
-      },
-      "Action": "sts:AssumeRoleWithWebIdentity",
-      "Condition": {
-        "StringEquals": {
-          "'$OIDC_PROVIDER_URL':aud": "sts.amazonaws.com",
-          "'$OIDC_PROVIDER_URL':sub": [
-            "system:serviceaccount:ack-system:ack-sagemaker-controller",
-            "system:serviceaccount:ack-system:ack-applicationautoscaling-controller"
-          ]
-        }
-      }
-    }
-  ]
-}
-' > ./trust.json
-```
-
-Updating an ApplicationAutoscaling ScalableTarget requires the following permissions. First create a service-linked role for ApplicationAutoscaling.
-
-```sh
-aws iam create-service-linked-role --aws-service-name sagemaker.application-autoscaling.amazonaws.com
-```
-
-Then create a file named pass_role_policy.json to create the policy required for the IAM role.
-
-```sh
-printf '{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Action": "iam:PassRole",
-      "Resource": "arn:aws:iam::'$AWS_ACCOUNT_ID':role/aws-service-role/sagemaker.application-autoscaling.amazonaws.com/AWSServiceRoleForApplicationAutoScaling_SageMakerEndpoint"
-    }
-  ]
-}
-' > ./pass_role_policy.json
-```
-
-Run the following command to create a role with the trust relationship defined in `trust.json`. This role enables the Amazon EKS cluster to get and refresh credentials from IAM.
-
-```sh
-OIDC_ROLE_NAME=ack-controller-role-$CLUSTER_NAME
+### 3.0 Samples 
```

Comment:
Does this need to be a separate section ? Maybe we can just combine these with the individual installation sections ?

Reply:
ok for now.